ui/src/components/Masthead.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { PageHeader, Toolbar, ToolbarGroup, ToolbarItem, Dropdown, DropdownToggle, DropdownItem } from '@patternfly/react-core';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { css } from '@patternfly/react-styles';
+import { withRouter } from 'react-router-dom';
+
+class Masthead extends React.Component {
```

Comment:
I have never heard this terminology used before, "Masthead". Is this an alternative name for the header/navigation header?

Reply:
@davidffrench It's patternfly terminology but I actually didn't mean to check this in. It should be part of the Header pr. Will remove it from here for now.